build.sbt

```diff
@@ -776,6 +777,20 @@ lazy val nativeBridge04 = project
     fork in Test := true
   )
 
+// Temporaryily publish a fork of sbt test-agent.
+// This version includes https://github.com/sbt/sbt/pull/5800
+// while we wait for the next release of sbt.
+lazy val testAgent = project
```

Comment:
Would it be simpler to manually publish the artifact from the `sbt` repo under a different orgzaniation (ch.epfl.scala) instead of creating a new project in the bloop build? When upstream sbt publishes a new version we only have to change the organization back. I can help publish the fork if you like, I got it working from my local computer the other day.

Reply:
Good point. Moreover, it appears there are more mysteries that I'd have to solve in the build, and it's unlikely we'll have more changes to make to the test agent. Thanks!